XPNet.GraphicsTestPlugin/GraphicsTestPlugin.cs

```diff
@@ -79,7 +114,9 @@ private int DrawingHook(XPLMDrawingPhase inPhase, int inIsBefore)
 
 			var (lat, lon, alt) = m_api.Graphics.LocalToWorld(res.LocationX, res.LocationY, res.LocationZ);
 
-			testTug.Draw(0, 0, new XPLMDrawInfo_t[] { new XPLMDrawInfo_t(res.LocationX, res.LocationY, res.LocationZ, 0.0f, 0.0f, 0.0f) });
+			m_testTug.Draw(0, 0, new XPLMDrawInfo_t[] { new XPLMDrawInfo_t(res.LocationX, res.LocationY, res.LocationZ, 0.0f, 0.0f, 0.0f) });
```

Comment:
I've done a little more reading about the Draw API, and I guess I'm onboard with the DrawInfo object being exposed directly, since you need to be able to efficiently pass an array of them in the draw loop, and it'd be better not to have an API that requires creating new CLR array objects for every invocation of the draw loop.  But if we're going to do that, I'd want to follow the notes I included above, naming it and its members per the public API naming conventions.  (That will work just fine, it won't break the underlying calls to X-Plane, b/c the C layer won't care what we named a field, just so long as we kept it in the right location in the struct).

Reply:
Done